src/Forms/Shared/Samples/Map/DownloadPreplannedMap/readme.md

```diff
@@ -43,8 +41,8 @@ The [Naperville stormwater network map](https://arcgisruntime.maps.arcgis.com/ho
 
 `PreplannedUpdateMode` can be used to set the way the preplanned map area receives updates in several ways:
 
-* `NoUpdates` - No updates will be performed.
-* `SyncWithFeatureServices` - Changes, including local edits, will be synced directly with the underlying feature services.
+* `NoUpdates` - No updates will be performed. This mode is intended for when a static snapshot of the data is required, and it does not create a replica. This is the mode used for this sample.
```

Comment:
Would it be worthwhile to specify 'on the service' re: replica? Intuitively the offline copy would be a replica of the online content, but I know 'replica' has a specific ArcGIS meaning.